src/tools/pkcs11-tool.c

```diff
@@ -5244,12 +5287,16 @@ static int encrypt_decrypt(CK_SESSION_HANDLE session,
 		CK_OBJECT_HANDLE privKeyObject)
 {
 	EVP_PKEY       *pkey;
-	unsigned char	orig_data[] = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', '\0'};
+	unsigned char	orig_data[512] = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', '\0'};
```

Comment:
I think using the rest of the 512 bytes will yield a warning of uninitialized data. You can use `pseudo_randomize` to initialize it properly for the test.

Reply:
Fixed by https://github.com/OpenSC/OpenSC/pull/1600/commits/b327b76134b19e2495e01abce47ca1aef2d32034